constants/chatbotCommands.py

```diff
@@ -1440,30 +1440,53 @@ async def editMap(fro: str, chan: str, message: list[str]) -> Optional[str]:
     for md5 in beatmap_md5s:
         await glob.redis.publish("cache:map_update", f"{md5['beatmap_md5']},{status}")
 
-    status_to_colour = lambda s: {5: 0xFF90EB, 2: 0x66E6FF, 0: 0x696969}[s]
+    # Map icons & misc
+    icon_akatsuki = "<:akatsuki:1160855094712078368>"
+    icon_chimu = "<:chimu:824401502830460958>"
+    icon_beatconnect = "<:beatconnect:1170350492967243899>"
+
+    # osu! game mode emoji dictionary
+    mode_to_emoji = lambda s: {
+        3: "<:modemania:1087863868782547014>",
+        2: "<:modefruits:1087863938982612994>",
+        1: "<:modetaiko:1087863916278853662>",
+        0: "<:modeosu:1087863892308410398>",
+    }[s]
+
+    # Colour & icons used in stable client
+    status_to_colour = lambda s: {5: 0xFF66AA, 2: 0x6BCEFF, 0: 0x696969}[s]
+    status_to_emoji_id = lambda s: {
+        5: "1166976753869279272",
+        2: "1166976760424964126",
+        0: "1166976756230651934",
+    }[s]
+
+    # Get previous status (for emojis too, pretty icons...)
+    prev_status_readable = status_to_readable(res["ranked"])
+    prev_status_emoji_id = status_to_emoji_id(res["ranked"])
 
     # Get the nominator profile URL just once
     nominator_profile_url = userUtils.getProfile(token["user_id"])
 
     webhook = discord.Webhook(
         url=settings.WEBHOOK_NOW_RANKED,
-        title=res["song_name"],
         colour=status_to_colour(status),
-        author=f"This {message[1]} has recieved a status update. :pencil:",
-        author_url=f'https://chimu.moe/d/{res["beatmapset_id"]}',
-        author_icon="https://cdn.discordapp.com/emojis/1160855094712078368.png",
-        image=f'https://assets.ppy.sh/beatmaps/{res["beatmapset_id"]}/covers/cover.jpg?1522396856',
+        author=f"{fro} ({token['user_id']})",
+        author_url=f"{nominator_profile_url}",
+        author_icon=f"https://a.akatsuki.gg/{token['user_id']}",
+        title=f'{mode_to_emoji(res["mode"])} {res["song_name"]}',
+        title_url=f'https://bathbot.de/osudirect/{res["beatmapset_id"]}',
```

Comment:
idk about bathbot as the download... is your intention to download an osz or to redirect to ingame direct? cus we can do both without needing bathbot

Reply:
we're redirecting it to ingame direct! afaik you can't use `osu://` links on the embed - if that was an option i think bathbot wouldn't have made that site.

Reply:
arghhhhhhhhhhh i hate more dependencies on random websites i kinda would rather it just not go to direct

Reply:
that's ok! let's just make that redirect to the official bancho listing then? i think that seems to make the most sense

Reply:
changed things in https://github.com/osuAkatsuki/bancho-service/pull/92/commits/459d1a09985d1e6500fcb075815eef46cf7313d6 to suggested changes

Reply:
@tsunyoku im going to sleep but i think everything looks good now, here's the current embed...

![](https://cdn.discordapp.com/attachments/1165545768917139536/1171197086709055578/image.png)